lib/oauth.rb

```diff
@@ -14,8 +15,10 @@ def description
     end
   end
 
-  def self.scopes(privileged: false)
-    scopes = SCOPES
+  def self.scopes(oauth2: false, privileged: false)
+    scopes = []
+    scopes += OAUTH2_SCOPES if oauth2
+    scopes += SCOPES
```

Comment:
I'd put `SCOPES` back first as in the old code, and move `OAUTH2_SCOPES` to the end for consistency with ordering in other places.